File: examples/FaxLineRemoveUserExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
)

with ApiClient(configuration) as api_client:
    fax_line_remove_user_request = models.FaxLineRemoveUserRequest(
        number="[FAX_NUMBER]",
        email_address="member@dropboxsign.com",
    )

    try:
        response = api.FaxLineApi(api_client).fax_line_remove_user(
            fax_line_remove_user_request=fax_line_remove_user_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling FaxLineApi#fax_line_remove_user: %s\n" % e)

File: examples/FaxLineRemoveUserExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();
apiCaller.username = "YOUR_API_KEY";

const faxLineRemoveUserRequest = new models.FaxLineRemoveUserRequest();
faxLineRemoveUserRequest.number = "[FAX_NUMBER]";
faxLineRemoveUserRequest.emailAddress = "member@dropboxsign.com";

apiCaller.faxLineRemoveUser(
  faxLineRemoveUserRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLineApi#faxLineRemoveUser:");
  console.log(error.body);
});
